Sources/ShortcutUI/SwiftUI/View+OnLoad.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ViewDidLoadModifier.swift
+//  ShortcutUI
+//
+//  Created by Swathi on 2022-06-09.
+//  Copyright © 2022 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct ViewDidLoadModifier: ViewModifier {
+
+    @State private var didLoad = false
+    private let action: (() -> Void)?
+
+    init(perform action: (() -> Void)? = nil) {
+        self.action = action
+    }
+
+    func body(content: Content) -> some View {
+        content.onAppear {
+            if !didLoad {
+                didLoad = true
+                action?()
+            }
+        }
+    }
+
```

Comment:
maybe remove this line?

Reply:
yes

Reply:
All the files in the library have a last empty line. It didnt complain in the swiftlint. Do we have an updated file?